src/app.js

```diff
@@ -1 +1,26 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+const { faker } = require('@faker-js/faker');
+
+const content = faker.lorem.word();
+const fileName = faker.lorem.word();
+const firstPath = `./src/${fileName}.txt`;
+const secondPath = `./src/${fileName} copy.txt`;
```

Comment:
why are you hardcoding this values? consider using `path` module or `process.argv`